students/ScottL/lesson09/Donors.py

```diff
@@ -0,0 +1,48 @@
+# ------------Donors.py Module ---------------#
+# Desc:  Classes for donor data
+# Dev:   Scott Luse
+# Date:  03/17/2018
+# ChangeLog:(When,Who,What)
+# ---------------------------------------------#
+if __name__ == "__main__":
+    raise Exception("This file is not meant to run by itself")
+
+import Reporting as rp
+
+class Donor_object(object):
+    '''Turns a dictionary into a class'''
+
+    # --Constructor--
+    def __init__(self, dictionary):
+        '''Constructor'''
+        for key in dictionary:
+            setattr(self, key, dictionary[key])
```

Comment:
maybe do a `self.donar_data = {setattr(self, key, dictionary[key]) for key in dictionary}

Then you can reference anything via `self.donar_data` instead of `for key in self.__dict__:` which is not really what you want to be doing